src/labels/CustomLabels.labels

```diff
@@ -7327,6 +7327,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Stay Track in GS for End Users</shortDescription>
+        <value>Learn more about Activities in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Stay Track in GS for End Users</shortDescription>
+        <value>Track Activities</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Stay Track in GS for End Users</shortDescription>
+        <value>Stay on Track with Activities</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemWorkMultipleRecordsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Work Multiple Records in GS for End Users</shortDescription>
+        <value>List views help you easily find what you need by providing filtered groups of Contacts, Accounts, Opportunities, or other objects.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemWorkMultipleRecordsLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Work Multiple Records in GS for End Users</shortDescription>
+        <value>Learn more about List Views in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemWorkMultipleRecordsPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Work Multiple Records in GS for End Users</shortDescription>
+        <value>Visualize Processes</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemWorkMultipleRecordsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Work Multiple Records in GS for End Users</shortDescription>
```

Comment:
I recommend changing this to: Title of Work with Multiple Records checklist item in Get Started for End Users.